FAQ.md

```diff
@@ -54,9 +54,9 @@ The commercial license can be provided for free if you help us promote Slint: Ch
 ### What are the commercial licensing options?
 
 We offer - Ambassador, Flex and Buyout commercial licensing options. All options
```

Comment:
Unrelated to this change: this section could benefit from an unordered list or markdown table.